O21.Game/Localization/Translations.fs

```diff
@@ -0,0 +1,16 @@
+module O21.Localization.Translations
+
+open FSharp.Data
+
+type Language = 
```

Comment:
Why do we have hard coded languages here? What about getting all the languages from the translations folder?

Reply:
I think that we can add "Language" key to json

Reply:
> Why do we have hard coded languages here? What about getting all the languages from the translations folder?

Meh, I'm not marking this as a requirement for now. But if we get a third language (and it is likely, in the future though), then we will.

> I think that we can add "Language" key to json

Same as the previous one.